cde-pentaho/resource/js/cdf-dd-base.js

```diff
@@ -269,7 +269,7 @@ var SynchronizeRequests = {
 
     $.post(wd.cde.endpoints.getPluginUrl() + "Syncronize", saveSettingsParams, function(result) {
       try {
-        var json = eval("(" + result + ")");
+        var json = JSON.parse(result);

```

Comment:
The code contained inside the *cde-pentaho* folder is used for the 4.x version which is no longer supported. @pamval should we be doing changes here, or should these be discarded and only be considered and tested if a backport to 4.x is needed?